cms/toolbar/toolbar.py

```diff
@@ -24,7 +23,7 @@
 
 
 class CMSToolbarLoginForm(AuthenticationForm):
-    username = forms.CharField(label=_("Username"), max_length=100)
+    username = forms.CharField(max_length=100)
```

Comment:
I assume we can completely remove this, it's already defined in ``AuthenticationForm``, right?